internal/rpm/rpm.go

```diff
@@ -64,6 +64,12 @@ func GetAllRPMs(ctx context.Context, root string) ([]Info, error) {
 		}
 		rpms = append(rpms, Info{Name: f[0], NVRA: f[1]})
 	}
+	if err := scanner.Err(); err != nil {
+		return nil, fmt.Errorf("error reading rpm -qa: %w", err)
+	}
+	if len(rpms) == 0 {
```

Comment:
I am thinking that if this scanner is to work for any image in the future then it should pass on a FROM scratch base image that has go binaries with no crypto symbols.

Reply:
Well, here we scan rpms (and ignore everything else), so if there are no rpms and as a result we scanned 0 files we should at least emit a warning.

Maybe we should rename this command from `scan node` to `scan rpmroot`, and add `scan node` command which will do walk over all files (same as we do for `scan payload`. WDYT?

Reply:
@kolyshkin Yes, something like that may make sense from a naming perspective.